File: java/src/CACC/week3/GuessNumber.java
package CACC.week3;

import java.util.Random;
import java.util.Scanner;

public class GuessNumber {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        Random random = new Random();
        System.out.println("Type in an integer as range");
        int range = scanner.nextInt();

//        int number;

        for ( int i=0; i < 50; i++) {
            int number = random.nextInt(range);
            System.out.println(number);
        }

        int number = random.nextInt(range);

        int guess;
        do {
            System.out.println("Guess an integer ");
            guess = scanner.nextInt();
            if (guess == number) {
                System.out.println("you got it right!");
            } else if (guess > number) {
                System.out.println("you got it too big!");
            } else {
                System.out.println("you got it too small!");
            }

        } while(guess != number);

    }
}

File: python/cacc/week5/guess_number.py
import random

r = int(input("Type in an integer as range: "))

target = random.randint(0, r)

while True:
    g = int(input("Input your guess: "))
    if g == target:
        print('you got it correct')
        break
    elif g < target:
        print('too small')
    else:
        print('too large')
